test/integration/TokenAssociateIntegrationTest.js

```diff
@@ -1,20 +1,27 @@
 import {
-    // AccountBalanceQuery,
-    // AccountCreateTransaction,
+    AccountAllowanceApproveTransaction,
+    AccountBalanceQuery,
+    AccountCreateTransaction,
+    AccountUpdateTransaction,
+    Hbar,
+    NftId,
     // AccountInfoQuery,
-    // Hbar,
-    // PrivateKey,
+    PrivateKey,
     Status,
     TokenAssociateTransaction,
     TokenCreateTransaction,
+    TokenMintTransaction,
+    TokenType,
+    TransactionId,
+    TransferTransaction,
 } from "../../src/exports.js";
 import IntegrationTestEnv from "./client/NodeIntegrationTestEnv.js";
 
 describe("TokenAssociate", function () {
     let env;
 
     before(async function () {
-        env = await IntegrationTestEnv.new();
+        env = await IntegrationTestEnv.new({ balance: 1000 });
```

Comment:
The default balance is 100 hbars. One token create is around 15 hbars so the transactions start to fail since this operator is used for every test case and the default balance is not enough. 
@svetoslav-nikol0v can give more insight on this